training_mod_consts/src/lib.rs

```diff
@@ -971,50 +945,104 @@ impl ToggleTrait for CharacterItem {
     }
 }
 
+bitflags! {
+    pub struct MashTrigger : u32 {
+        const HIT =            0b0000_0000_0000_0000_0001;

```

Comment:
Why not follow `1 << 0` etc?

Reply:
Can't the default be more easily written as `MashTrigger::HIT || MashTrigger::BLOCK || ...`? I would much rather use the enum itself, imagine if we reordered the bits and didn't change the default.

Reply:
Unfortunately not. That was what I tried at first actually, but Rust complained about non-const functions being run during initialization of the static mut.